.gitignore

```
@@ -0,0 +1,2 @@
+.env
+.idea
```

```suggestion
.idea
```

Developer-specific code should be defined in a [global gitignore file](https://sebastiandedeyne.com/setting-up-a-global-gitignore-file/) instead of at the repository level.

---

This would cut across all projects. Added to an IDEAS ticket to discuss https://collaboration.homeoffice.gov.uk/jira/browse/HOCS-2742